sdks/python/standard_expansion_services.yaml

```diff
@@ -0,0 +1,104 @@
+# generated wrapper class name.
+#
+# Transform identifiers listed in the `ignore` field will be skipped.
+
+- gradle_target: 'sdks:java:io:expansion-service:shadowJar'
+  destinations:
+    python: 'apache_beam/io'
+  transforms:
+    'beam:schematransform:org.apache.beam:kafka_write:v1':
+      name: 'WriteToKafka'
+      destinations:
+        python: 'apache_beam/io/kafka'
+    'beam:schematransform:org.apache.beam:kafka_read:v1':
+      name: 'ReadFromKafka'
+      destinations:
+        python: 'apache_beam/io/kafka'
+
+- gradle_target: 'sdks:java:io:google-cloud-platform:expansion-service:shadowJar'
+  destinations:
+    python: 'apache_beam/io/gcp'
+  transforms:
+    'beam:schematransform:org.apache.beam:bigquery_storage_write:v2':
+      name: 'StorageWriteToBigQuery'
+      destinations:
+        python: 'apache_beam/io/gcp/bigquery'
+    'beam:schematransform:org.apache.beam:bigquery_storage_read:v1':
+      name: 'StorageReadFromBigQuery'
+      destinations:
+        python: 'apache_beam/io/gcp/bigquery'
+    'beam:schematransform:org.apache.beam:bigquery_fileloads_write:v1':
+      name: 'FileLoadsToBigQuery'
+      destinations:
+        python: 'apache_beam/io/gcp/bigquery'
+    'beam:schematransform:org.apache.beam:bigquery_export_read:v1':
+      name: 'ExportReadFromBigQuery'
+      destinations:
+        python: 'apache_beam/io/gcp/bigquery'
+    'beam:schematransform:org.apache.beam:bigtable_write:v1':
+      name: 'WriteToBigtable'
+      destinations:
+        python: 'apache_beam/io/gcp/bigtable'
+    'beam:schematransform:org.apache.beam:bigtable_read:v1':
+      name: 'ReadFromBigtable'
+      destinations:
+        python: 'apache_beam/io/gcp/bigtable'
+    'beam:schematransform:org.apache.beam:pubsub_read:v1':
+      name: 'ReadFromPubSub'
+      destinations:
+        python: 'apache_beam/io/gcp/pubsub'
+    'beam:schematransform:org.apache.beam:pubsub_write:v1':
+      name: 'WriteToPubSub'
+      destinations:
+        python: 'apache_beam/io/gcp/pubsub'
+    'beam:schematransform:org.apache.beam:pubsublite_read:v1':
+      name: 'ReadFromPubSubLite'
+      destinations:
+        python: 'apache_beam/io/gcp/pubsublite'
+    'beam:schematransform:org.apache.beam:pubsublite_write:v1':
+      name: 'WriteToPubSubLite'
+      destinations:
+        python: 'apache_beam/io/gcp/pubsublite'
+    'beam:schematransform:org.apache.beam:spanner_cdc_read:v1':
+      name: 'ReadFromSpannerChangeStreams'
+      destinations:
+        python: 'apache_beam/io/gcp/spanner'
+    'beam:schematransform:org.apache.beam:spanner_write:v1':
+      name: 'WriteToSpanner'
+      destinations:
+        python: 'apache_beam/io/gcp/spanner'
+    'beam:schematransform:org.apache.beam:jdbc_write:v1':
+      name: 'WriteToJdbc'
+      destinations:
+        python: 'apache_beam/io/jdbc'
+    'beam:schematransform:org.apache.beam:jdbc_read:v1':
+      name: 'ReadFromJdbc'
+      destinations:
+        python: 'apache_beam/io/jdbc'
+  ignore:
```

Comment:
should we exclude hand-written wrappers for now? We now have two ReadFromKafka (and possibly other) transforms.

Reply:
+1